### DISCUSSION TITLE
Changing Theme for btop with the change in theme for hyprland.

### DISCUSSION THREAD

**Discussion by Exoutia** [prasanthrangan/hyprdots#384] [General]
I want to change the theme of btop with the change of theme in hyprland How should I do it.
Just giving me a direction about how changing theme work for different application and as btop is a terminal application file i should be able to copy those steps.

**Comment by prasanthrangan** [comment]
Hi, this should be very simple to implement with symlink -
1. create config files for each theme you have configured, example `Catppuccin-Mocha.conf`, `Catppuccin-Latte.conf`...
2. create `theme.conf` symlink file and link it to current theme
3. create `btop.conf` and source the `theme.conf` file within
4. update `themeswitch.sh` to sync the `theme.conf` symlink file to current theme, similar to kitty (refer this as an example) -
https://github.com/prasanthrangan/hyprdots/blob/4451b1df3c2f34817c1cb16d740422e332ceb330/Configs/.config/hypr/scripts/themeswitch.sh#L92

**Reply by Exoutia** [reply]
Thanks I will do that and get back to you if I get any problem ☺️

**Reply by prasanthrangan** [reply]
Anytime.. happy to help!
